src/theory/quantifiers/sygus/term_database_sygus.cpp

```diff
@@ -1572,11 +1572,6 @@ Node TermDbSygus::evaluateBuiltin(TypeNode tn,
     }
     if (!res.isNull())
     {
-      Assert(res
```

Comment:
Why is this assertion removed?

Reply:
The assertion is now in the evaluator itself, so that we don't have to have it at every call site: https://github.com/CVC4/CVC4/pull/2171/files#diff-09e6de4f37c686bc6c75e235aa393660R148

Reply:
Sounds good